Assets/Scripts/Robot/Sonar.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+public class Sonar : MonoBehaviour
+{
+    //Script counts all distances from objects in tasks
+
+    public float viewRadious;
```

Comment:
Should use private instead of public and use [SerializedField] to show variable in Unity Inspector